src/Juvix/Compiler/Nockma/Translation/FromTree.hs

```diff
@@ -987,45 +961,43 @@ runCompilerWith _opts constrs moduleFuns mainFun =
     mainClosure :: Term Natural
     mainClosure = makeMainFunction (runCompilerFunction compilerCtx mainFun)
 
+    -- This term is not quoted and cannot be evaluated.
     funcsLib :: Term Natural
     funcsLib = Str.theFunctionsLibrary @ makeList compiledFuns
       where
         compiledFuns :: [Term Natural]
         compiledFuns =
-          (OpQuote # (666 :: Natural)) -- TODO we have this unused term so that indices match. Remove it and adjust as needed
+          (nockNilTagged "anomaGetPlaceholder")
+            : (nockNilTagged "mainFunctionPlaceholder")
```

Comment:
Was the `666` placeholder previously for the `main` function that is added to `allFuns` but not `libFuns`? 

The `libFunShift` is required because `anomaGet` is now added to the functions library?

Reply:
I'd just like to clarify why `libFunShift` is `1` but 2 extra functions are being added to the front of the functions library. We should comment this while we remember it.